t/api/14-plugin_obs_rsync.t

```diff
@@ -0,0 +1,68 @@
+# Copyright (C) 2019 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+use Mojo::Base -strict;
+
+BEGIN {
+    unshift @INC, 'lib';
+}
+
+use FindBin;
+use lib "$FindBin::Bin/../lib";
+use Test::More;
+use Test::Mojo;
+use OpenQA::Test::Database;
+use Mojo::File qw(tempdir path);
+
+# we must create db, otherwise OpenQA::WebAPI will try to (unused in test)
+my $db = OpenQA::Test::Database->new->create(skip_fixtures => 1);
+
+# this test also serves to test plugin loading via config file
+my @conf = (
+    "[global]\n", "plugins=ObsRsync\n", "[obs_rsync]\n",
+    "home=" . Mojo::File->new(__FILE__)->dirname->dirname->child('data')->child('openqa-trigger-from-obs') . "\n"
+);
+my $tempdir = tempdir;
+$ENV{OPENQA_CONFIG} = $tempdir;
+path($ENV{OPENQA_CONFIG})->make_path->child("openqa.ini")->spurt(@conf);
```

Comment:
I don't think the `make_path` call here does anything, since `tempdir` already creates the directory.